### PR TITLE
[VEN-2696] Payments

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN-2696

### CODE REVIEW

File: vips/vip-344/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+      {
+        target: VTREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [WBNB, COMMUNITY_BNB_AMOUNT.add(SKYNET_BNB_AMOUNT), NORMAL_TIMELOCK],
+      },
+      {
+        target: WBNB,
+        signature: "approve(address,uint256)",
+        params: [WBNB, COMMUNITY_BNB_AMOUNT.add(SKYNET_BNB_AMOUNT)],
+      },
```
Comment (chechu) [inline vips/vip-344/bscmainnet.ts:92]: Is this mandatory?

File: vips/vip-344/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDC, COMMUNITY_USDC_AMOUNT, COMMUNITY],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "_grantXVS(address,uint256)",
+        params: [SKYNET, SKYNET_XVS_AMOUNT],
+      },
+      {
+        target: COMMUNITY,
```
Comment (chechu) [inline vips/vip-344/bscmainnet.ts:120]:
```suggestion
        target: SKYNET,
```

**Review by chechu — APPROVED** [review]

File: vips/vip-346/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    version: "v2",
+    title: "VIP-346 Payments issuance for audits and other expenses",
+    description: `#### Summary
+
+If passed this VIP will perform the following actions:
+
+- Transfer 35,000 USDT to Certik for the retainer program (July and August 2024)
+- Transfer 7,500 USDT to Fairyproof for the audit of Two Kinks Interest Rates
+- Transfer 21,600 USDC to Chainalysis for the yearly subscription fee renewal
+- Transfer 170,000 USDT to ChaosLabs for the Quarterly fee
```
Comment (kkirka) [inline vips/vip-346/bscmainnet.ts:54]:
```suggestion
- Transfer 170,000 USDC to ChaosLabs for the Quarterly fee
```

Comment (chechu) [reply vips/vip-346/bscmainnet.ts]: 1549acc6f3f3387d4352183ee1f05af74f35c445

**Review by kkirka — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/346?chainId=56

**Pull request merged**